akka-docs/src/test/scala/docs/stream/IntegrationDocSpec.scala

```diff
@@ -195,6 +197,62 @@ class IntegrationDocSpec extends AkkaSpec(IntegrationDocSpec.config) {
     probe.expectMsg("akkateam@somewhere.com")
   }
 
+  "actorRefWithAck" in {
+    //#actorRefWithAck
+    val words: Source[String, NotUsed] =
+      Source(List("hello", "hi"))
+
+    // sent from actor to stream to "ack" processing of given element
+    val AckMessage = AckingReceiver.Ack
+
+    // sent from stream to actor to indicate start, end or failure of stream:
+    val InitMessage = AckingReceiver.StreamInitialized
```

Comment:
I'd be OK with skipping those `val`s and adding the comments when passing the parameters

Reply:
+1